packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -142,7 +142,7 @@ hub.post("/", async (req, res) => {
       });
       const rejectedRetryResults = await Promise.allSettled(rejectedRetryPromiseArray);
       rejectedRetryResults.forEach((result, index) => {
-        _processSpokeResponse(Object.keys(configObject)[index], result, validOutputs, errorOutputs);
+        _processSpokeResponse(retriedOutputs[index], result, validOutputs, errorOutputs);
```

Comment:
this was the main bug in the severless hub

Reply:
aha nice catch